tools/run_tests/build_artifact_python.sh

```diff
@@ -81,5 +81,19 @@ then
   done
 fi
 
+# We need to use the built grpcio-tools/grpcio to compile the health proto
+# Wheels are not supported by setup_requires/dependency_links, so we
+# manually install the dependency
+if [ "$SKIP_PIP_INSTALL" == "" ]
```

Comment:
I wanted to avoid install grpcio/grpcio-tools if we are not in a virtualenv or docker, so I ended up using this variable to enforce that.  I've renamed it to BUILD_HEALTH_CHECKING